### ISSUE TITLE
Add pdb files to nuspec

### ISSUE THREAD

**Issue by imanushin** [gsscoder/commandline#424]
During code analys it is better to have ability to decompile code to understand, what it does.
In this case it is better to add pdb files to final nuget package.

Additionally we will have ability to debug this code.

[MSVS can debug external code with known PDBs files.](https://stackoverflow.com/questions/654696/how-to-debug-external-class-library-projects-in-visual-studio)

**Comment by phatcher** [comment]
Can I also suggest that you use pdbgit as well as this would link the pdb symbols to the Github source

**Comment by imanushin** [comment]
@phatcher , this is nice tool, however, I'd like to keep pdbs with dll files in the same package, because:
1. It allow me to see source code without it downloading
2. I will see detailed callstack in exception on each computer, even it is disconnected from internet.

As I understand, pdbgit just simplify debugging.

Also: what's @gsscoder 's decigion?

**Comment by phatcher** [comment]
@imanushin, I don't think this would be an issue for you - if you have the correct version locally and navigate to it then you will resolve to local sources.

All pdbgit does is change the source path embedded in the pdb to point at the github hash version rather than the local path that the published version was compiled on.

**Comment by imanushin** [comment]
Nice, thank you! It is interesting.

Will wait until this my current pull request will be pushed. Then will add this extension.
